assets/scripts/admin.js

```diff
@@ -0,0 +1,122 @@
+(function($) {
+  function renderMediaUploader(variation_id) {
```

Comment:
Is this custom code or taken over from another plugin?

Reply:
This is custom code.